subsys/bluetooth/controller/hci/hci.c

```diff
@@ -3065,7 +3097,7 @@ static void le_tx_test(struct net_buf *buf, struct net_buf **evt)
 	uint8_t status;
 
 	status = ll_test_tx(cmd->tx_ch, cmd->test_data_len, cmd->pkt_payload,
-			    0x01);
+			    0x01, 0, 0, 0, NULL, BT_HCI_TX_TEST_POWER_MAX_SET);
```

Comment:
We aim to do not use magic numbers everywhere it is possible. That eases reading and understanding a code. E.g. `cte_length` argument may use `BT_HCI_LE_NO_CTE_TX` instead of `0`.

Reply:
I'm already using `BT_HCI_LE_TEST_CTE_DISABLED` for cte_length.

Reply:
Done, all magic numbers are replaced by macros